public/locales/it/translation.json

```diff
@@ -0,0 +1,59 @@
+{
+  "about": "Più informazioni su di noi",
+  "available": "Disponibile",
+  "blog": "Articoli del blog",
+  "community": "Comunità",
+  "contributors": "Contributori",
+  "countryCode": "it",
+  "description": "Verifica la disponibilità dei nomi che vuoi dare a i tuoi progetti  in tutti i registri principali contemporaneamente.",
+  "gettingStarted": "Inizia",
+  "gettingStartedWithExample": "Prova con namae",
+  "issues": "Utilizzi",
```

Comment:
Is this the correct translation for (GitHub) Issues?

Reply:
Yes, yes

Reply:
Hmm. Quick dictionary lookup leads me to 'Uses'. Can you provide some Italian blog posts that include the translated word for 'GitHub Issues'? Just to clarify.

Reply:
Ah, I was wrong now I correct with Problemi